### ISSUE TITLE
About Test data of JRBD dataset

### ISSUE THREAD

**Issue by wisionpeng** [vita-epfl/social-transmotion#11]
The project uses ndjson files obtained through the processing method from the JRDB project's source code for training. However, the JRDB dataset website provides only unlabeled test data. How can ground truth data for the testing process be obtained?

**Comment by Fujiry0** [comment]
They split the scenes of the training set of the original JRDB dataset into train, val and, test set for their own split.

**Comment by yanggao2000** [comment]
Hi, @Fujiry0 thanks for the explanation! To further clarify, we select the static-camera scenes from the training split, and our detailed splits are attached in the screenshot below. You can also find our processed data in the latest release 'checkpoint and data'.
<img width="239" alt="Screenshot 2024-10-25 at 11 27 28" src="https://github.com/user-attachments/assets/b2a8cd5a-3077-4edf-b1b9-8b4c26c11200">
